tests/visualization_tests/matplotlib_tests/test_slice.py

```diff
@@ -21,33 +23,47 @@ def test_plot_slice() -> None:
     # Test with no trial.
     study = prepare_study_with_trials(no_trials=True)
     figure = plot_slice(study)
-    assert len(figure.get_lines()) == 0
+    assert len(figure.findobj(PathCollection)) == 0
 
     study = prepare_study_with_trials(with_c_d=False)
 
     # Test with a trial.
     figure = plot_slice(study)
     assert len(figure) == 2
-    assert len(figure[0].get_lines()) == 0
-    assert len(figure[1].get_lines()) == 0
+    assert len(figure[0].findobj(PathCollection)) == 1
+    assert len(figure[1].findobj(PathCollection)) == 1
     assert figure[0].yaxis.label.get_text() == "Objective Value"
 
+    # Scatter plot data is available as PathCollection
```

Comment:
To be consistent with other comments, please put a period at the end of a sentence.
```suggestion
    # Scatter plot data is available as PathCollection.
```